dev-requirements.txt

```diff
@@ -1,4 +1,6 @@
 black==22.6.0
+dbt-postgres==1.4.1
```

Comment:
I think we want to either let the developer install dbt and an adapter on their own, or let this install latest and have developers override the version. Server v0.2.0 will only be used with 1.5+, so at the very least we'd want to install `1.5.0b1` here and use the `1.4.1` installation on the `0.1.latest` branch.

Alternatively, we could have a wrapper for the `pytest` command for local dev that accepts a dbt version and installs it and the adapters, or defaults to a dbt version expected to work with that server version

Reply:
Offline chatted with Rachel, we tended to
- Keep core and adaptors as external dependency, hence I removed adaptor dependency.
- However some e2e tests require such dependency, added skip rule to skip those tests.